File: client/src/app/shared/data-models/payitem.model.ts
import { Injectable } from "@angular/core";

@Injectable({
    providedIn: 'root'
})
export class PayItemModel {
    id: String = "";
    organizationId: String = "";
    itemName: String = "";
    description: String = "";
    itemType: String = "";
    paymentType: String = "";
    status: String = "";
}

File: src/main/java/com/hris/HRIS/model/PayItemModel.java
package com.hris.HRIS.model;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Getter
@Setter
@ToString

@Document(collection = "payitem")
public class PayItemModel {
    @Id
    private String id;
    private String organizationId;
    private String itemName;
    private String description;
    private String itemType;
    private String paymentType;
    private String status;
}
